Parsers/ASimAuditEvent/Parsers/ASimAuditEventVectraCloudAudit.yaml

```diff
@@ -0,0 +1,52 @@
+Parser:
+  Title: Audit Event ASIM parser for VectraCloud Audit Logs Event
+  Version: '0.1'
+  LastUpdated: Mar 17 2023
+Product:
+  Name: Vectra
+Normalization:
+  Schema: AuditEvent
+  Version: '0.1.0'
+References:
+- Title: ASIM Audit Event Schema
+  Link: https://aka.ms/ASimAuditEventDoc
+- Title: ASIM
+  Link: https://aka.ms/AboutASIM
+Description: |
+  This ASIM parser supports normalizing VectraCloud Audit Logs Event in the Audits_Data_CL table to the ASIM Audit Event schema.
+ParserName: ASimAuditEventVectraCloudAudit
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let parser = (disabled:bool = false)
+  {
+    Audits_Data_CL
+    | extend
+          EventEndTime = event_timestamp_t,
+          EventProduct = 'Vectra Cloud',
+          EventSchema = "AuditEvent",
+          EventSchemaVersion = "0.1.0",
+          EventStartTime = event_timestamp_t,
+          EventType = "Other",
+          EventVendor = 'Vectra',
+          Type = "Audit Log",
+          EventUid = tostring(toint(id_d)),
+          ActorUserId = tostring(toint(user_id_d)),
+          ActorUserIdType = "UID",
+          ActorUsernameType = "UPN",
+          EventResult = case(result_status_s=="success", "Success", result_status_s=="failure", "Failure","NA")
+    | project-rename
+          Dvc = source_ip_s,
```

Comment:
Are we sure that source_ip_s is not the IP of the actor and is the Dvc IP Address?
If yes, DvcIpAddr should be populated and Dvc should be the Alias

Reply:
@devikamehra We have mapped the source_ip_s with the DvcIpAddr field and additionally, we have also added Dvc column that corresponds to the DvcIpAddr field.